independent-projects/arc/tests/src/test/java/io/quarkus/arc/test/contexts/request/propagation/RequestContextPropagationTest.java

```diff
@@ -41,8 +42,9 @@ public void testPropagation() {
         assertNotNull(controller2.getButton());
         assertTrue(controller2.getButton() == controller1.getButton());
 
-        // Store existing instances
-        ContextState state = requestContext.getState();
+        // capture state - two active beans
```

Comment:
What does "two active beans" mean? There is only one `SuperController` instance stored in the request context.